tests/system/providers/amazon/aws/example_dynamodb_to_s3.py

```diff
@@ -51,6 +55,8 @@
 @tenacity.retry(
     stop=tenacity.stop_after_attempt(5),
     wait=tenacity.wait_exponential(min=5),
+    before=before_log(log, logging.INFO),
+    before_sleep=before_sleep_log(log, logging.WARNING),
```

Comment:
Why WARN for one and INFO for the other?

Reply:
`before` is before an attempt so nothing anormal here. `before_sleep` happens between a fail and a sleep, hence the warn here :) [See doc](https://tenacity.readthedocs.io/en/latest/#before-and-after-retry-and-logging)

Reply:
I guess in my head I would have only warned if it times out, but I guess you could argue that should be ERROR at that point, so this looks good to me!